README.org

```diff
@@ -5,19 +5,24 @@
 
 * Orgzly Revived
 
-Orgzly is an outliner for taking notes and managing to-do lists.
+Orgzly Revived is a community-maintained version of Orgzly, as the development of the original app is no longer active.
 
-You can keep notebooks stored in plain-text and have them synchronized
-with a directory on your mobile device, SD card, WebDAV server or Dropbox.
+Orgzly Revived project aims to continue development of the Orgzly application, adding useful features, 
```

Comment:
```suggestion
Orgzly Revived aims to continue development of the Orgzly application, adding useful features, 
```

Reply:
See the reworded version. Here I actually did specifically mean the project, the Github organization, if you will. That is the "community" behind, as opposed to the app itself. Even though it is currently pretty-much the same thing.